membrane_curvature/base.py

```diff
@@ -0,0 +1,68 @@
+import numpy as np
+
+from membrane_curvature.surface import get_z_surface
+from membrane_curvature.curvature import mean_curvature, gaussian_curvature
+
+
+from MDAnalysis.analysis.base import AnalysisBase
+import MDAnalysis as mda
+
+
+class MembraneCurvature(AnalysisBase):
+
+    def __init__(self, universe, select='all',
+                 n_x_bins=100, n_y_bins=100,
+                 x_range=None,
+                 y_range=None,
+                 pbc=True, **kwargs):
+
+        super().__init__(universe.universe.trajectory, **kwargs)
+        self.selection = universe.select_atoms(select)
```

Comment:
Just to make everything clearer to future developers I would rename this from self.selection to either self.ag or self.atomgroup. That way folks can look at the name and immediately know what type of object it is (also better matches other MDA analysis methods).